floortrans/loaders/mm_dataset.py

```diff
@@ -0,0 +1,42 @@
+import pickle
+
+import torch
+import numpy as np
+import lmdb
+from torch.utils.data import Dataset
+
+
+class MMDataset(Dataset):
+    _lmdb_env = None
+
+    def __init__(self, db_path, dataset, augmentations):
+        if not MMDataset._lmdb_env:
+            MMDataset._lmdb_env = lmdb.open(
```

Comment:
Noob python question incoming:
_lmdb_env is a static member variable right?
If that is true and __init__ is called twice with different db_path, the second one will use the same db_path as the first

Reply:
Yes, on subsequent calls the db_path will be ignored, since the object already exists.

Reply:
I think that will be a bit non-obvious to anyone actually using this with two different db_paths, but OK.